packages/atlas/codegen.config.ts

```diff
@@ -2,7 +2,9 @@ import { CodegenConfig } from '@graphql-codegen/cli'
 
 import { customSchemaLoader } from './scripts/customSchemaLoader'
 
-const schemaUrl = 'https://orion.gleev.xyz/graphql'
+const ENV = process.env.VITE_DEFAULT_DATA_ENV?.toUpperCase() ?? 'PRODUCTION'
```

Comment:
for someone `VITE_DEFAULT_DATA_ENV` might be empty even if he's using the development build, so I would make a fallback variable of `VITE_ENV` before hardcoded production